### ISSUE TITLE
Artisan Install not working

### ISSUE THREAD

**Issue by mapodev** [thedevdojo/voyager#2499]
- Laravel Version: 5.5.30
- Voyager Version: ^1.0
- PHP Version: 7.1.5
- Database Driver & Version: mysql

### Description:

I created a brand new Laravel instance. Installed things with composer install, did composer require tcg/voyager. Set up the database configs for connection and did the "php artisan voyager:install", but I am getting this:

```
Setting up the hooks
Publishing complete.
Hooks are now ready to use! Go ahead and try to "php artisan hook:install test-hook"

In Connection.php line 664:

  SQLSTATE[42S02]: Base table or view not found: 1146 Table 'd029120b.menus' doesn't exist (SQL: select * from `menus` where `name` = admin limit 1)


In PDOConnection.php line 79:

  SQLSTATE[42S02]: Base table or view not found: 1146 Table 'd029120b.menus' doesn't exist


In PDOConnection.php line 77:

  SQLSTATE[42S02]: Base table or view not found: 1146 Table 'd029120b.menus' doesn't exist

```

**Comment by mapodev** [comment]
Ok, fixed it by following this comment: https://github.com/the-control-group/voyager/issues/901#issuecomment-291470960

**Comment by marktopper** [comment]
This can also be an side effect of the new release of `larapack/voyager-hooks` which might require the moving of [this line](Yea, it just requires the move of this line to later in the Voyager install progress:
https://github.com/the-control-group/voyager/pull/2496/files/69787b68f18591965d59a9a99446933ebca5c7d6#diff-57ffd60d2a32bd7a4bdbdf1e08e47147L69) to later in the install progress.

A temporary fix would be to run `composer require larapack/voyager-hooks:v1.0.4`.
> I recommend removing the version specification after installation.

**Comment by marktopper** [comment]
Also fixed in #2496
